src/Uno.CrossTargetting.props

```diff
@@ -61,8 +61,8 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'uap10.0' ">
-	<TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
-	<TargetPlatformMinVersion>10.0.14393.0</TargetPlatformMinVersion>
+	<TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
```

Comment:
shouldn't we go for the more current one? 17763.

See umbrella PRs : https://nventive.visualstudio.com/Umbrella/_git/Umbrella.View/pullrequest/19858?_a=files&path=%2Fbuild%2FUmbrella.View.CrossTargeting.props

Reply:
Right, let's sync.